DEPS

```diff
@@ -56,7 +56,7 @@ vars = {
   # Dart is: https://github.com/dart-lang/sdk/blob/main/DEPS
   # You can use //tools/dart/create_updated_flutter_deps.py to produce
   # updated revision list of existing dependencies.
-  'dart_revision': '1bf43bfd314768e235f6e5618842469bab6494bd',
+  'dart_revision': 'eb89827cad4b04c8c31a3fc1cbaa66af1c4b7998',
```

Comment:
Shouldn't this be a stable dart release?

Looks like a beta: https://github.com/dart-lang/sdk/commit/eb89827cad4b04c8c31a3fc1cbaa66af1c4b7998